frontend/public/extend/devconsole/components/pipelines/PipelineVisualizationTask.scss

```
@@ -0,0 +1,118 @@
+$border-color: var(--pf-global--BorderColor--light);
+$gutter: 1.8em;
+
+.odc-pipeline__task {
+  position: relative;
+  width: 13em;
+  height: 2em;
+  white-space: normal;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: #fafafa;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  margin-top: 1.5em;
+  padding: 0.5em;
+
+  &.input-node {
```

Technically also an element that shares similarities as the `__task`
```suggestion
  &__input-node {
```
or could just make it a modifier:
```suggestion
  &.is-input-node {
```